index.html

```diff
@@ -207,6 +250,9 @@ <h3 class="Skill">Skills</h3>
                     <div class="text_area">
                         <textarea maxlength="500" id="msg" name="user_message" placeholder="Enter text here" class="text_box" required></textarea>
                     </div>
+                    <!-- <div class="text_area-desktop">
+                        <textarea maxlength="500" id="msg" name="user_message" placeholder="Hi Mariana Glad to see you" class="text_box" required></textarea>
+                    </div> -->
```

Comment:
- [ ] It is important to note that unnecessary commented-out codes should be removed from your codebase as they make your codebase unorganized and difficult to understand. Please remove these commented codes and others in both HTML and CSS page 😃 👍